prince-of-versions/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="prince_of_versions_shared_preference_file">co.infinum.princeofversions.PREF_FILE</string>
```

Comment:
What's the actual benefit of having this constant in resource instead of having it as constant in the class file?

Reply:
There isn't any benefit in this case I don't think.